configs/bedrock.conf

```diff
@@ -0,0 +1,23 @@
+# Bedrock default config file.
+# Run `bedrock -?` for help with command line arguments
+
+# Path of db file; will be created if absent
+BEDROCK_DB_PATH="/var/lib/bedrock/bedrock.db"
+
+# Single node, unknown environment
+THISNODE="bedrock"
+PRIORITY=200
+
+# Cache size in kilobytes
+CACHE_SIZE="10001"
+
+VERBOSE="-v"
+
+# Bind to network address
+SERVER_HOST="0.0.0.0:8888"
+NODE_HOST="0.0.0.0:8889"
+
+#EXTRA_FLAGS="-readThreads 1 -plugins db,jobs,cache,mysql"
```

Comment:
This won't work, you can only have a value stored in these variables, not full flags and values.

Additionally, readThreads is deprecated, workerThreads should be used.

Reply:
>This won't work, you can only have a value stored in these variables, not full flags and values.

Are you saying this doesn't follow some convention?  Because it definitely works.